### ISSUE TITLE
Compatibility with @ngxs ^18.0.0

### ISSUE THREAD

**Issue by davemoser** [ngxs-labs/actions-executing#22]
Hello there 👋 

This package is not compatible with the new @ngxs/store@18.x.y releases. I guess it's mostly about the versions referenced in `package.json`.

Is there any plan on updating this package to make it work with the newest ngxs versions again sometime soon? If there is no capacity I can also try to do it myself and create a pull request.

**Comment by joaqcid** [comment]
hi @davemoser yes, definitely is on the plans, if you could create a PR aligning version would be great, ill try to get to this this week

**Comment by joaqcid** [comment]
hi @davemoser  v18 is release, can you please check if its working now?

**Comment by davemoser** [comment]
Thank you very much @joaqcid ! Works perfect so far on our side. I had some issues setting the repo up locally because I am not used to yarn. Have been using npm all the time. But I got it to build now.
I also already tested it with my Angular 18 App and there it also works well! 👍 

Maybe a sidenote that the Demo [Link](https://ngxs-labs-actions-executing.netlify.com/) on the Readme is not working anymore.

Will close my issue as completed. Thanks again 😄
